flybywire-aircraft-a320-neo/ModelBehaviorDefs/A32NX/AirlinerCommon.xml

```diff
@@ -430,7 +431,7 @@
                     <FREQUENCY>3</FREQUENCY>
                     <UPDATE_CODE>
                     (A:COM TRANSMIT:#Radio_ID#, Bool) if{
-                        #Button_ID# (&gt;L:XMLVAR_COM_Transmit_Channel)
+                        <!-- #Button_ID# (&gt;L:XMLVAR_COM_PANEL#ID#_Transmit_Channel) -->
```

Comment:
I was not able to spot the purpose for these lines of code. To me, it was part of all the problems. 

Thefore, just in case for the future, I left the condition only.